styleguide/derek/global-components/tracking/tracking.js

```diff
@@ -0,0 +1,190 @@
+/* eslint no-use-before-define: 0 */
+
+const Zoolander = Zoolander || {};
+const dataLayer = {};
```

Comment:
^^ also, we need to use the existing `dataLayer` if present as with `Zoolander` above.